ci-operator/step-registry/assisted/baremetal/operator/capi/assisted-baremetal-operator-capi-ref.yaml

```diff
@@ -18,5 +18,7 @@ ref:
   dependencies:
   - name: "hypershift-operator"
     env: HYPERSHIFT_IMAGE
+  - name: "cluster-api-provider-agent"
+    env: PROVIDER_IMAGE
```

Comment:
Is it used in this step? I thought we set it up in ``assisted-baremetal-setup-commands.sh`` and merge it into ``/root/config``: https://github.com/openshift/release/blob/08e7671ec8e7ee3db6bbc04ed573a01996d186cb/ci-operator/step-registry/assisted/baremetal/setup/assisted-baremetal-setup-commands.sh#L130

In ``assisted-baremetal-operator-capi-commands.sh`` I couldn't find a reference of ``PROVIDER_IMAGE`` (and sourcing ``/root/config`` should get this either way)

Reply:
Removed this part for now, 
It seems that we need to add export of the imagesed deps in assisted-baremetal-operator-capi-commands.shed